docs/cpp/guides/flags.md

```diff
@@ -125,6 +125,10 @@ ABSL_FLAG(std::vector<std::string>, languages,
           std::vector<std::string>({"english", "french", "german"}),
           "comma-separated list of languages to offer in the 'lang' menu");
 ABSL_FLAG(absl::Duration, timeout, absl::Seconds(30), "Default RPC deadline");
+ABSL_FLAG(std::optional<std::string>, image_file, std::nullopt,
+          "Sets the image input from a file.");
+// An optional flag can also indicate its valueless state with empty braces {}
```

Comment:
While true, this seems like a strange thing to point out. The empty braces aren't specific to ABSL_FLAG in any way, but rather a valid way to construct a std::optional is any (well, most) contexts.